etcd_test.go

```diff
@@ -0,0 +1,126 @@
+package dsync
+
+import (
+	"fmt"
+	"github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
+	"os"
+	"testing"
+	"time"
+)
+
+var (
+	etcd client.KeysAPI
+)
+
+func init() {
+	node := os.Getenv("ETCD_NODE")
+	if node == "" {
+		node = "http://localhost:2379"
+	}
+	cfg := client.Config{Endpoints: []string{node}}
```

Comment:
I prefer line spacing between statements that work together. The if is a statement and the creating of the config value is starting a new block or idea.